python/bigipconfigdriver.py

```diff
@@ -80,38 +178,38 @@ class K8sCloudBigIP(CloudBigIP):
       BigIP partition
     - For each backend (task, node, or pod), it creates a pool member and adds
       the member to the pool
-    - If the app has a Marathon Health Monitor configured, create a
-      corresponding health monitor for the BigIP pool member
     - Token-based authentication is used by specifying a token named 'tmos'.
       This will allow non-admin users to use the API (BIG-IP must configure
       the accounts with proper permissions, for either local or remote auth).
 
     Args:
-        hostname: IP address of BIG-IP
-        username: BIG-IP username
-        password: BIG-IP password
-        partitions: List of BIG-IP partitions to manage
+        bigip: ManagementRoot object
+        partition: BIG-IP partition to manage
+        schema_path: Path to the CCCL schema
     """
 
-    def __init__(self, hostname, port, username, password, partitions,
-                 manage_types):
-        """Initialize the K8sCloudBigIP object."""
-        super(K8sCloudBigIP, self).__init__(hostname, port, username,
-                                            password, partitions,
-                                            token="tmos",
-                                            manage_types=manage_types)
+    def __init__(self, bigip, partition, schema_path, manage_types):
```

Comment:
I have looked into this in more detail, and while CCCL still has some of the manage_types code in it still, it has no effect. Therefore in this file manage_types should be removed from this code altogether (and eventually from CCCL).